docs/source/general/importing_users.rst

```diff
@@ -0,0 +1,79 @@
+===============
+Importing users
+===============
+
+This feature can be used for importing users from a csv file. There are many features included in it such as:
+
+* Importing users in batches: all of the users of a particular csv file would be stored in batches and can be retrieved/ deleted easily using the batch functions.
+* Set an expiration date: Expiration date can be set for a batch after which the users would not able to authenticate to the RADIUS Server.
+* Autogenerate usernames and passwords: The usernames and passwords are automatically generated if they aren't provided in the csv file. Usernames are generated from the email address whereas passwords are generated randomly and their lengths can be customized.
+* Passwords are accepted in both cleartext and hash formats from the CSV.
+* Send mails to users whose passwords have been generated automatically.
+
+It can be done using both a management command and the admin interface.
+
+``batch_add_users``
+-------------------
+
+This command imports users from a csv file. Usage is as shown below.
+
+.. code-block:: shell
+
+    ./manage.py batch_add_users --name <name_of_batch> \
+                                --file <filepath> \
+                                --expiration <expiration_date> \
+                                --password-length <password_length>
+
+Note that the expiration and password-length are optional parameters which default to never and 8 respectively.
+
+Using the admin interface
+-------------------------
+
+Selecting the CSV as the strategy and uploading the CSV file is all one will have to do to import the CSV file from the admin interface. It can be checked at `/admin/radiusbatch/add`.
```

Comment:
could you add one or two screenshots or a brief gif for this? I think it's useful for users to see it in action.